package/MDAnalysis/core/topologyattrs.py

```diff
@@ -516,24 +516,20 @@ def center_of_mass(group, **kwargs):
             If ``True``, move all atoms within the primary unit cell before
             calculation. [``False``]
 
-        .. note::
+        Returns
+        -------
+        center : ndarray
+            center of group given masses as weights
+
+        Notes
+        -----
             The :class:`MDAnalysis.core.flags` flag *use_pbc* when set to
             ``True`` allows the *pbc* flag to be used by default.
 
         .. versionchanged:: 0.8 Added `pbc` parameter
         """
-        atomgroup = group.atoms
-
-        pbc = kwargs.pop('pbc', flags['use_pbc'])
-        masses = atomgroup.masses
-
-        if pbc:
-            positions = atomgroup.pack_into_box(inplace=False)
-        else:
-            positions = atomgroup.positions
-
-        return np.sum(positions * masses[:, np.newaxis],
-                      axis=0) / masses.sum()
+        return group.atoms.center(weights=group.atoms.masses,
```

Comment:
I haven't don't done any checks. But I assume it is a tiny bit slower now. This is because numpy average does the same thing but with a few more sanity checks. I can check with some benchmarks. I think we don't pay much for single calls but calling this function a lot now might lead to slightly lower performance.